### ISSUE TITLE
Caldav Could not fetch Calendars

### ISSUE THREAD

**Issue by PWNL506** [elementary/switchboard-plug-onlineaccounts#237]
### What Happened?

When i try to connect Caldav Calendar ID, I get a warning "Could not fetch calendars, Failed to get propertie: HTTP error code 401 (unauthorised)
what do i do?

### Steps to Reproduce

1. Go to Online Accounts
2. Add CalDAV
3. Insert needed Information

### Expected Behavior

It should have synced my calendar

### OS Version

6.x (Odin)

### Software Version

Latest release (I have run all updates)

**Comment by selwynorren** [comment]
I get the same issue. I cannot load Caldav server at all. I do use Google Worskpace so I am not sure if this is a problem. I do have 2FA on and I am use specific app password as well.

**Comment by selwynorren** [comment]
Hi All,

Curious if there is any progress on this?

**Comment by selwynorren** [comment]
Hey Everyone, I am eager to try out the brand new release of eOS 7. Has this been fixed yet by any chnace?

**Comment by stan-janssen** [comment]
I am succesfully using CalDAV on eOS 7 (I was on eOS 6 as well, by the way). This is with a "regular", self-hosted caldav server ([Radicale](https://radicale.org/)).

**Comment by selwynorren** [comment]
Yeah I am pretty sure with a self hosted option it would work very well. However I make use of Google Workspace so I have no desire or need to use anything other than Google for my contact, calendar and would love the task integration to work properly.

This way I can sync my information no matter whether using my desktop or laptop. But it need to be Google
